Run MilCodeGen for PresentationCore

ThomasGoulet73 opened the pull request:
Contributes to dotnet/wpf#10429

## Description
Generates the code in PresentationCore using the changes from dotnet/wpf#10430.

I recommend reviewing only the last commit and with whitespace changes hidden, use [this link](https://github.com/dotnet/wpf/pull/10434/commits/8f44ef015a0d68c8fc4d351972fc0ad3cade0125?diff=unified&w=1).

## Customer Impact
None, shouldn't affect customers.

## Regression
No.

## Testing
Local build + CI + I manually validated the generated code.

## Risk
Low, there shouldn't be any functional changes.
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/wpf/pull/10434)

ThomasGoulet73 commented:
This PR is blocked by dotnet/wpf#10430.

h3xds1nz commented:
That is to reduce the diff. `TextDecorationCollection` is like one of the few that got re-formatted because most of them are actually as `[ index ]`.

dipeshmsft commented:
> TextDecorationCollection is like one of the few that got re-formatted because most of them are actually as [ index ].

I thought this was reverted back to this state because of the limitation of the generator. This code gets generated from this place : 
https://github.com/dotnet/wpf/blob/5f713e22cf3f6800277ec0acb5a4a03cbc8b096e/src/Microsoft.DotNet.Wpf/src/WpfGfx/codegen/mcg/helpers/CollectionHelper.cs#L192
and if we remove the spaces, the generator crashes throwing a syntax error.

h3xds1nz commented:
Eitherway is possible tbh, I know where its coming from but I didn't try to modify it.

ThomasGoulet73 commented:
I did hit the bracket limitation but the main reason I reverted `TextDecorationCollection` instead of fixing the code generator was because, as @h3xds1nz said, it was the only 1 of many files formatted manually so either I reverted this file or changed a bunch of other files and I chose the option with the smaller diff.

> and if we remove the spaces, the generator crashes throwing a syntax error.

I hit this problem also but there's way to workaround this limitation, you just need to concat the string with the `[` and `]` in a variable instead of inline and it will fix the parser error.

I do plan on working on formatting the generated code in the future.
